Tests/Components Tests/Gift Card/GiftCardComponentTests.swift

```diff
@@ -489,7 +489,7 @@ class GiftCardComponentTests: XCTestCase {
 
         UIApplication.shared.keyWindow?.rootViewController = sut.viewController
 
-        wait(for: .milliseconds(300))
+        wait(for: .milliseconds(50))
```

Comment:
Just a thought - should we parametrise this?

```swift
let aMoment = DispatchTimeInterval.milliseconds(50)
wait(for: aMoment)
```

Reply:
Sounds good - could be an extension of DispatchTimeInterval `wait(for: .aMoment)`
Was also thinking about abstracting it into something like this:
```
func setupRootViewController(_ viewController: UIViewController) {
    UIApplication.shared.keyWindow?.rootViewController = viewController
    UIApplication.shared.keyWindow?.layer.speed = 10.0 // 10x Faster animations
    wait(for: .aMoment) // Giving the viewController a moment to get presented
}
```

```